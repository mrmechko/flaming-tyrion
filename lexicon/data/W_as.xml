<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="../style/lookup.xsl"?>
<!DOCTYPE WORD SYSTEM "word.dtd">
<WORD name="as" modified="Sun Mar  5 01:43:27 2017">
<POS name="adv">
<CLASS onttype="event-time-rel" words="after,afterward,afterwards,ago,anytime,as,as quick as possible,as soon as,as soon as possible,awhile,before,earlier,early,following,forthcoming,from,immediately,in progress,in the meantime,late,later,nowadays,once,one day,pending,previously,prior to,recently,right away,simultaneously,since,sometime,soon,then,til,till,to,today,tomorrow,tonight,underway,until,upcoming,when,whenever,while,yesterday,yet" ancestors="temporal-location,temporal-modifier,temporal-predicate,predicate,abstract-object,referential-sem,any-sem,root">
<FRAME desc="binary-constraint-s-decl-templ" example="as this happens the valve closes" />
</CLASS>
<CLASS onttype="manner" words="all right,allright,alright,alternately,altogether,apart,as,as if,as though,competently,directly,fast,hard,in,o k,ok,okay,polite,separately,well,with" ancestors="situation-modifier,situation-object-modifier,predicate,abstract-object,referential-sem,any-sem,root">
<FRAME desc="binary-constraint-s-or-np-templ" example="use/try/select/find/ this as an example" />
</CLASS>
<CLASS onttype="reason" words="as,because,in response to,since" ancestors="situation-modifier,situation-object-modifier,predicate,abstract-object,referential-sem,any-sem,root">
<FRAME desc="binary-constraint-s-decl-templ" />
</CLASS>
</POS>
<POS name="prep">
<CLASS onttype="as" words="" ancestors="">
<FRAME desc="no-features-templ" />
</CLASS>
</POS>
</WORD>
