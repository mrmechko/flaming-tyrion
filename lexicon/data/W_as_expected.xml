<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="../style/lookup.xsl"?>
<!DOCTYPE WORD SYSTEM "word.dtd">
<WORD name="as_expected" modified="Sat May 13 01:45:06 2017">
<POS name="adv">
<CLASS onttype="predicate" words="anyhow,as expected" ancestors="abstract-object,referential-sem,any-sem,root">
<FRAME desc="disc-templ" />
</CLASS>
</POS>
</WORD>
